jmeter-service/main.go

```diff
@@ -204,12 +204,16 @@ func runWorkload(serviceUrl *url.URL, testInfo *TestInfo, id string, workload *W
 		return true, nil
 	}
 
-	os.RemoveAll(workload.TestStrategy + "_" + testInfo.Service)
-	os.RemoveAll(workload.TestStrategy + "_" + testInfo.Service + "_result.tlf")
+	// the resultdirectory is unique as it contains context but also gives some human readable context such as teststrategy and service
+	// this will also be used for TSN parameter
+	resultDirectory := fmt.Sprintf("%s_%s_%s", workload.TestStrategy, testInfo.Service, testInfo.Context)
```

Comment:
Thinking of troubleshooting with multiple projects, services, and stages, would it make sense to also use the project name and stage here?
E.g., 
```
fmt.Sprintf("%s_%s_%s_%s_%s", testInfo.Project, testInfo.Service, testInfo.Stage, workload.TestStrategy, testInfo.Context)
```

Reply:
yeah - thats a valid point. I thought that test strategy and service would be unique. but - adding all these makes sense. just want to make sure we dont run itno any directory length restrictions